### ISSUE TITLE
Not able to link BrowserView accessibility tree to the main accessibility tree

### ISSUE THREAD

**Issue by sarthak-saxena** [electron/electron#26305]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Problem description
I have a use case where I have embedded a browser view inside browser window. Since browser view is not part of the DOM hierarchy hence the accessibility tree is not linked to the main window(parent browser window).

So my use case is to link the browser view accessibility tree with the main window accessibility tree **such that screen readers(JAWS/NVDA) are able to enumerate all the possible landmarks for navigation**

Please check this repository for source code to reproduce the issue
https://github.com/sarthak-saxena/electron-browserview-issue

Please also find attached a video in which we can see that keyboard navigation is not moving focus to browser view since it is not part of the main BrowserWindow accessibility tree.
![Browser view not part of ax tree (2)](https://user-images.githubusercontent.com/9450414/98878358-c994a600-24a8-11eb-8993-eb3dc0364b03.gif)

### Expected Behavior
There should be a way to link the accessibility tree of browser view to the main AX tree.

### Actual Behavior
No way to link the accessibility tree of browser view.

**Comment by sarthak-saxena** [comment]
@codebytere I have attached a link to the repository for source code to reproduce the issue
https://github.com/sarthak-saxena/electron-browserview-issue

**Comment by codebytere** [comment]
Some digging into this unfortunately confirms that this isn't possible given that way that `BrowserView`s are implemented by design. `WebViews` are part of the primary `webContents` even though you can for example open devtools for the webview only - this is because we attach the `webView` to the primary `webContents` as an iframe. BrowserViews, while they are intended to be an alternative to `WebViews`, are isolated views by contrast - they're not linked into the primary `webContents`.  You have to navigate between the two views in order to trigger accessible tab navigation for the webContents of each. Screen readers should themselves be able to handle this.

**Comment by sarthak-saxena** [comment]
See, the accessibility tree of BrowserView when embedded inside a BrowserWindow is separate & is not linked. 
So, if we focus on the main BrowserWindow then while enumerating all possible landmark heading or links on the screen reader, it shows only for  BrowserWindow & not the embedded BrowserView 
& similarly, if we focus BrowserView embedded inside of BrowserWindow, it shows enumeration only for the BrowserView.

So our use case here is when the BrowserWindow is focussed which has BrowserView embedded inside that, then the landmark enumeration for links or headings should show all the possible headings or links on that page. 
Basically, the problem here is that while enumeration heading or links of BrowserView is not getting counted in which I believe should be fixed.
Please check the video(with audio) for more clarity on the issue
https://www.loom.com/share/5ffb738b77f7472d8039a4c66a2c905e
